### DISCUSSION TITLE
A way to get the Prometheus config (like `/config`)

### DISCUSSION THREAD

**Discussion by tanguynicolas** [prometheus-operator/prometheus-operator#7046] [Q&A]
Hi,

Is there a way to get the Prometheus configuration? Like when going to the `/config` HTTP path. _Or maybe a way to get the list of targets._

Didn't found in the documentation. I need this for a particular usecase. 

Something like `prom.config()`, as can be done to retrieve the list of metrics with `prom.all_metrics()`.

Thanks

**Comment by slashpai** [comment]
May be you can use API provided by Prometheus https://prometheus.io/docs/prometheus/latest/querying/api/?

**Reply by tanguynicolas** [reply]
I'm so confused! I thought I had posted on the `4n4nd/prometheus-api-client-python` repo!

Sorry for the mistake. And thank you very much for your reply, which is still relevant and partly answers my question. I'll be able to get by with these endpoints if the Pyhton client doesn't allow them to be queried.

These endpoints will do the job:

`https://prometheus.demo.do.prometheus.io/api/v1/targets`
`https://prometheus.demo.do.prometheus.io/api/v1/status/config`
